modules/ROOT/assets/attachments/sg.yaml

```diff
@@ -73,6 +73,61 @@ properties:
             description: |
               Configures how long Sync Gateway should wait for a bucket operation to complete before timing out and trying again. The value can be increased in scenarios where there is a heavy load on Couchbase Server and operations are likely to take more than 2.5 seconds to complete. The default value is 2500 milliseconds.
             default: 2500
+          delta_sync:
+            type: object
+            description: |
+              *NOTE:* Delta Sync is an Enterprise Edition feature on Sync Gateway and Couchbase Lite.
+
+              Delta Sync is the ability to replicate only parts of the Couchbase mobile document that have changed. This can result in significant savings in bandwidth consumption as well as throughput improvements, especially when network bandwidth is typically constrained.
```

Comment:
I'm wondering if we should call out the additional bucket storage requirements when enabling delta sync. These can be tuned with the `rev_max_age_seconds` config option, as described further below, but is not mentioned up here in the general description of the feature.

Reply:
https://github.com/couchbase/docs-sync-gateway/commit/b316f29a3da260cf3599cb71f0474196ac6047ba